### PR TITLE
Privly Posting Button added to posting process

### PR DESCRIPTION
**Pull request by sammyshj**
Changes made in the PR:
* Added posting_button.js content script
* Modified posting_process.js background script to support privly posting button functionality

### CODE REVIEW

File: privly.safariextension/scripts/content_scripts/posting_button.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+/*global privlyUrlReceiptNode:false, pendingPost:false, safari:false, ls:true,  */
```
Comment (smcgregor) [inline privly.safariextension/scripts/content_scripts/posting_button.js]: Needs @fileOverview

Comment (sammyshj) [inline privly.safariextension/scripts/content_scripts/posting_button.js]: Added @fileOverview

Comment (smcgregor) [inline privly.safariextension/scripts/content_scripts/posting_button.js]: Almost there, you have a small typo, "...the it is displayed...".

Comment (sammyshj) [inline privly.safariextension/scripts/content_scripts/posting_button.js]: Fixed the comment.

**Comment by smcgregor** [comment]
There is still an issue with login. I can only successfully login if I visit privlyalpha.org, login, then return to the safari-extension:// context. I think this problem is unrelated to this pull request, so I will likely merge this in a moment.

**Pull request merged**